ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1081,22 +1081,23 @@ open class TableViewCell: UITableViewCell {
         }
     }
 
-    @available(*, deprecated, message: "Any color or stylistic changes on TableViewCell labels should be done through NSAttributedString.")
+    // Any color or stylistic changes on TableViewCell labels should be done through NSAttributedString
```

Comment:
Should we just be taking the color they give here and just applying it to the correct attributed string?

Reply:
I don't think so, I think we want to deter clients from using this in the future so eventually, and hopefully, we'll be able to remove this?